tests/apps/actorapp/app.go

```diff
@@ -22,7 +22,7 @@ const appPort = 3000
 
 const actorMethodURLFormat = "http://localhost:3500/v1.0/actors/%s/%s/method/%s"
 
-const registedActorType = "testactor" // Actor type.
+const registedActorType = "testactor" // Actor type must be unique per test app.
 const actorIdleTimeout = "5s"         // Short idle timeout.
 const actorScanInterval = "1s"        // Smaller then actorIdleTimeout and short for speedy test.
 const drainOngoingCallTimeout = "1s"
```

Comment:
Nit: why not use multiple line syntax for constants ? We used this style in the runtime codes.

e.g.
```
const (
   ...
   registedActorType = "testactor" // Actor type must be unique per test app.
   actorIdleTimeout = "5s"         // Short idle timeout.
   ...
)

Reply:
Sure, I was not aware of that. Thanks.